binddump.c

```diff
@@ -60,7 +60,7 @@ void free_dyld_cmd(void **dyld_cmd) {
 
 void free_dyld_array(dyld_t *dylds) {
 	size_t arr_size = sizeof(dylds)/sizeof(dylds[0]);
-	for(int i = 0; i < arr_size; i++) {
+	for(int i = 0; (size_t)i < arr_size; i++) {
```

Comment:
The thing about C is that all arithmetic types convert implicitly. A compiler may throw a warning up but the conversion is still correct.

That being said, `size_t` is an `unsigned` type so the type of `i` (could/should) be changed to an `unsigned` type.  I have been using `uintptr_t` in this project because it deals with files and `uintptr_t`  is the size of a pointer on a given system.